src/commands/build/index.ts

```diff
@@ -73,6 +73,11 @@ function printBuildResults(
   }
 
   for (const [dnpName, { releaseMultiHash }] of Object.entries(buildResults)) {
+    if (!releaseMultiHash) {
```

Comment:
why would a release hash not exist? this situation must never happen

Reply:
The type needs to be `string | undefined` because it will be undefined until the package is built and uploaded to an IPFS node